### DISCUSSION TITLE
Schema model

### DISCUSSION THREAD

**Discussion by maxwww** [abdukhashimov/student_aggregator#47] [General]
To parse users' files the system should know which schema should be used.
Storing Schemas in mongoDB can be  implemented in different ways (or more):
1) Storing in a separate collection without linking to User Model
User {id: 1, name: "name", email: "emai@ts.ts"...}
Schema {id: 1, name: "name", version: "1", fields: []...}
2) Storing in a separate collection with linking to User Model
User {id: 1, name: "name", email: "emai@ts.ts", schemas: [1]...}
Schema {id: 1, name: "name", version: "1", fields: []...}
3) Embedding Schema to User Model
User {id: 1, name: "name", email: "emai@ts.ts", schemas: [{id: 1, name: "name", version: "1", fields: []...}]...}

Now it implemented in the first way.
But let's find the best option.
Team, please share your thoughts.

**Comment by abdukhashimov** [comment]
I also prefer the first way

**Reply by abdukhashimov** [reply]
Just because it is easier to test and develop

**Comment by ihoro-epam** [comment]
OK, just for ease of development lets take first approach. We will be able to change it later.
